src/frontend/src/sass/_default.scss

```diff
@@ -1,12 +1,61 @@
-:root {
-  box-sizing: border-box;
-}
+  @include breakpoint(0 map-get($grid-breakpoints, 'md')) {
+  :root {
+    box-sizing: border-box;
+    --custom-font-size: 14px;
+  }
+  }
+  @include breakpoint(map-get($grid-breakpoints, 'md')) {
+  :root {
+    box-sizing: border-box;
+    --custom-font-size: 16px;
+  }
+  }
 
 body {
   min-height: 100vh;
   position: relative;
   font-family: $font-family;
-  font-size: 16px;
+  font-size: var(--custom-font-size);
+  @include breakpoint(0 map-get($grid-breakpoints, 'md')) {
+    /* converting to em as the :root is default to 16px
+    most likely set by bootstrap */
+    h1 {
+      font-size: 2.05em;
```

Comment:
bootstraps root variables are overwriting our :root variables unless i define custom variables. Don't want to spend too much time in modifying specificity.

Reply:
I feel that this could be an actual use case for the`!important` rule. Appending that to all of the default font-sizing will allow our font-sizing to overwrite bootstraps.